contributor-docs/code-owners.md

```diff
@@ -0,0 +1,9 @@
+# Code Owners
+
+Dataflow Templates uses [Code Owners](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)
+to manage code review. This means that for some PRs, reviewers will be automatically assigned.
+
+The goal is for this to eventually be fully automated, it is not yet, however,
+for certain directories. If you create a PR and a reviewer is not assigned
```

Comment:
can we add the link to the ownerfile?
https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/main/.github/CODEOWNERS